### DISCUSSION TITLE
Uncaught Error: Call to undefined function PrivateBin\parse_ini_file

### DISCUSSION THREAD

**Discussion by Anagastes** [PrivateBin/PrivateBin#1474] [Q&A] [answered]
### Describe the problem/question

I am currently doing a completely fresh installation.

As soon as I create a config.php I get this error on the page.

```
 Fatal error: Uncaught Error: Call to undefined function PrivateBin\parse_ini_file() in /opt/www/PrivateBin/lib/Configuration.php:136 Stack trace: #0 /opt/www/PrivateBin/lib/Controller.php(177): PrivateBin\Configuration->__construct() #1 /opt/www/PrivateBin/lib/Controller.php(130): PrivateBin\Controller->_init() #2 /opt/www/PrivateBin/index.php(24): PrivateBin\Controller->__construct() #3 {main} thrown in /opt/www/PrivateBin/lib/Configuration.php on line 136
```

Even with this default content.

```
;<?php http_response_code(403); /*
; config file for PrivateBin
;
; An explanation of each setting can be find online at https://github.com/PrivateBin/PrivateBin/wiki/Configuration.

[main]
; (optional) set a project name to be displayed on the website
; name = "PrivateBin"
```

### Did you use the FAQ section?

- [X] Yes, I have read [the FAQ](https://github.com/PrivateBin/PrivateBin/wiki/FAQ) and I found no solution/answer there.

### What you did?

1. Permissions checked
2. Owner checked

### Server OS

Debian 12

### Webserver

NGINX

### PrivateBin version

1.7.5

### Browser and version

Firefox latest

### Local operating system and version

EndeavourOS

### Issue reproducibility

No, I cannot reproduce it on https://privatebin.net.

**Comment by elrido** [comment, marked as answer]
[parse_ini_file](https://www.php.net/manual/en/function.parse-ini-file.php) is a PHP builtin function, which I'd assumed is required, since PHP uses it to parse it's own configuration file(s) `php.ini`. Maybe it was configured in the [disable-functions](https://www.php.net/manual/en/ini.core.php#ini.disable-functions)?

**Reply by Anagastes** [reply]
Point landing! It was disabled... T__T
